### ISSUE TITLE
Could not resolve dependencies for project: Could not find artifact org.apache.shardingsphere:shardingsphere-test-util:jar:5.5.0

### ISSUE THREAD

**Issue by gaolove** [apache/shardingsphere#32256]
## Question

 <dependency>
            <groupId>org.apache.shardingsphere</groupId>
            <artifactId>shardingsphere-jdbc</artifactId>
            <version>5.5.0</version>
 </dependency>

When I run mvn install I got this error:
**Could not resolve dependencies for project : Could not find artifact org.apache.shardingsphere:shardingsphere-test-util:jar:5.5.0 in central**

And I can‘t find shardingsphere-test-util On https://mvnrepository.com/search?q=shardingsphere-test-util

**Comment by terrymanu** [comment]
The CI is not break.
FIY: https://github.com/apache/shardingsphere/actions

**Comment by linghengqian** [comment]
- Duplicate with #31128 . Fixes by #31143 .
- For Maven.
```xml
<dependency>
        <groupId>org.apache.shardingsphere</groupId>
        <artifactId>shardingsphere-jdbc</artifactId>
        <version>5.5.0</version>
        <exclusions>
            <exclusion>
                <groupId>org.apache.shardingsphere</groupId>
                <artifactId>shardingsphere-test-util</artifactId>
            </exclusion>
        </exclusions>
</dependency>
```
- For Gradle.
```groovy
repositories {
    mavenCentral()
}

dependencies {
    implementation('org.apache.shardingsphere:shardingsphere-jdbc:5.5.0') {
        exclude group: "org.apache.shardingsphere", module: "shardingsphere-test-util"
    }
}
```
